src/mempool.rs

```diff
@@ -150,21 +199,168 @@ impl Mempool {
         }
     }
 
+    async fn on_sync_reply(
+        &mut self,
+        validator: &ValidatorPublicKey,
+        missing_cars: Vec<Car>,
+    ) -> Result<()> {
+        info!("{} SyncReply from sender {validator}", self.storage.id);
+
+        info!(
```

Comment:
same, I didn't saw this in my quick test, but I am asking myself if those won't be better in debug ?